config/initializers/slowweb.rb

```diff
@@ -1,4 +1,4 @@
 require 'slowweb'
 # todo: there is a bug where this is only called half as many times
-limit = ENV.fetch('OSCN_REQUESTS_PER_MINUTE', 6)*2
+limit = ENV.fetch('OSCN_REQUESTS_PER_MINUTE', 12).to_i*2
```

Comment:
slow web globally throttles all requests to specified APIs
This code checks for an environment variable named OSCN_REQUESTS_PER_MINUTE and defaults it to 12 if not provided  
I added to .to_i to force the variable to be an integer because environment variables actually always come in as strings which was causing an issue.

Reply:
Fabulous 👍